web/src/test/manual/web/chirality/chirality.js

```diff
@@ -10,6 +10,48 @@
  */
 
 function Keyboard_chirality() {
+  // Refer to $KEYMAN_ROOT/common/web/keyboard-processor/src/text/codes.ts, same method name.
+  // May be moved within common/web/types at some point in the future.
+  var getModifierState = function(layerId) {
```

Comment:
This method [defined within the `Codes` object (within this feature branch)](https://github.com/keymanapp/keyman/blob/77b3b8aecdce0ec3799a5c075b29e8b22ec959f1/common/web/keyboard-processor/src/text/codes.ts#L108-L148) no longer has an accessible established endpoint within KMW, and there's no way to import it from its module at runtime.

Rather than add another endpoint to manage longterm, I figured I'd just wholesale copy the method into this test keyboard.